dallinger/heroku/tools.py

```diff
@@ -164,7 +164,16 @@ def name(self):
 
     @property
```

Comment:
Maybe this should be a `cached_property`?

Reply:
+1 done